src/main/java/christmas/domain/calendar/Day.java

```diff
@@ -0,0 +1,20 @@
+package christmas.domain.calendar;
+
+import static christmas.message.ErrorMessages.INVALID_DATE_RANGE;
+
+public record Day(int day) {
+
+    private static int MIN_DAY = 1;
+    private static int EVEN_MONTHS_MAX_DAY = 31;
+
+    public Day {
+        validateRangeFromDay(day);
+    }
+
+    private void validateRangeFromDay(int day) {
+        if (day < MIN_DAY || day > EVEN_MONTHS_MAX_DAY) {
+            throw new IllegalArgumentException(INVALID_DATE_RANGE.getMessage());
```

Comment:
Exception을 enum으로 관리해도 좋을 것 같습니다.

Reply:
Enum에 예외타입을 정의하라는 의미인 것 같아서
 제 생각을 공유해보려고 합니다.

```suggestion
//.. ErrorMessages
public IllegalArgumentException getException() {
        return new IllegalArgumentException(message);
    }
```
제안해주신것과 같이 Enum에 예외 타입을 정의하게 된다면,
하나로 묶을 수 있는 장점을 가지긴 하지만,
예외 사항이 많아지고 책임질 것이 많아지면 
Enum 클래스가 커지게 될 것 같다는 의견입니다.

그래서 현재 저는 예외메시지만 Enum으로 정의하고 있습니다.

이 후 확장하게 된다면, 예외 메시지만 Enum으로 정의하고
클래스 상속을 이용하여 CustomException 타입을 정의하여 
사용하는것이 가독성과 유연성을 가질 것 같아요.

따라서 Enum은 상수 값을 나타내는데 주로 사용되기 때문에, 
예외와 관련된 Exception은 예외 클래스 자체가 책임지도록 하도록 하고,
메시지는 Enum으로 관리하는것이 좋다고 생각하여 
묶지 않고, 현재코드처럼 작성했습니다. :)

Reply:
아 맞네요. 과제 한정해서 Enum 으로 예외 처리하는 게 편해서 사용하다 보니 예외 책임에 대해 생각을 못 했습니다!
Exeption을 커스텀해서 사용하는 게 더 좋아 보이네요! :)